src/omd.mli

```diff
@@ -59,11 +59,14 @@ and block_desc =
   | Link_def of string link_def
   | Definition_list of def_elt list
 
-type doc = block list
+type doc = Ast.block list
 (** A markdown document *)
 
+module Html = Html
```

Comment:
If you need to access the definitions from another module, we should repeat the definitions here. The existence the `Html` submodule is a an implementation detail in my mind which may contain other stuff apart from that that we want to expose. The `Omd` interface is the public API of the library and so should stand on its own.